apps/angular/injection-token/src/app/timer.token.ts

```diff
@@ -0,0 +1,10 @@
+import { InjectionToken } from '@angular/core';
+
+export const TIMER = new InjectionToken<number>('timer');
```

Comment:
you can create a default factory function, this way you don't have to provide the timer if you use the default value.